packages/components/src/theme/theme-spectrum/theme-spectrum-overrides.css

```diff
@@ -9,6 +9,12 @@ label[class*='spectrum-'] {
   margin-bottom: 0;
 }
 
+[class*='spectrum-Menu-itemGrid'] svg,
```

Comment:
you could consider linking this to PR that would allow this lines removal

Reply:
Added